<launch>
	<!-- Gimbal control Node -->
	<node pkg="dji_gimbal_control" type="dji_gimbal_control_node" name="dji_gimbal_control" output="screen">
		<param name="track_tag" type="bool" value="true"/>
		<param name="camera_info_topic" type="str" value="/jetyak_uav_vision/dji_camera/camera_info"/>
		<param name="tag_pose_topic" type="str" value="/jetyak_uav_vision/ar_pose_marker"/>
		<param name="yaw_axis" type="int" value="0"/>
		<param name="pitch_axis" type="int" value="4"/>
		<param name="roll_axis" type="int" value="3"/>
		<param name="reset_angle_btn" type="int" value="0"/>
		<param name="face_down_btn" type="int" value="2"/>
		<param name="toggle_track_btn" type="int" value="3"/>
		<param name="Kp" type="double" value=".002"/>
		<param name="Kd" type="double" value="0.0005"/>
	</node>
</launch>
